deeplearning4j-core/src/main/java/org/deeplearning4j/nn/multilayer/MultiLayerNetwork.java

```diff
@@ -847,7 +847,7 @@ public INDArray params(boolean backwardOnly) {
         List<INDArray> params = new ArrayList<>();
         for (Layer layer: getLayers()){
             INDArray layerParams;
-            if( layer instanceof BasePretrainNetwork && backwardOnly)
+            if(layer instanceof BasePretrainNetwork)
                 layerParams = ((BasePretrainNetwork) layer).paramsBackprop();
```

Comment:
@nyghtowl if you look above in the same method you will find `if(backwardOnly) return params();	` which means `if( layer instanceof BasePretrainNetwork && backwardOnly)` will be executed only if `backwardOnly` was `false` which will make the whole expression always `false`!